src/main.js

```diff
@@ -7,6 +9,12 @@ import App from './App.vue';
 import {routes} from '@pages';
 import './icons';
 
+Sentry.init({
+  dsn:
+    'https://f92b29c910ce42189733ccdb2600dff2@o418830.ingest.sentry.io/5324513',
```

Comment:
environment variables D: VUE_APP_SENTRY (I'll add this for u to the appropriate places)